README.md

```diff
@@ -1,12 +1,9 @@
 fundamentals-ts
 ===============
 
-The draft C++ Library Fundamentals Technical Specification
+The draft C++ Library Fundamentals Technical Specification.
 
-Visit [the Polymer-based rendered
-version](https://rawgit.com/cplusplus/fundamentals-ts/v3/main.html)
-or the [standalone
-version](https://rawgit.com/cplusplus/fundamentals-ts/v3/fundamentals-ts.html).
+The draft can be viewed online at https://cplusplus.github.io/fundamentals-ts/
```

Comment:
This seems to be missing puncutation: either have a colon after "at", or a full stop at the end.

Reply:
A full stop would probably get (harmlessly) treated as part of the URL when converting markdown to HTML, so I added a colon.

Reply:
I understand the concern, though I've never seen a 21st century autolinker that would make such a mistake...